Ports/libzip/patches/0001-Disable-some-unneeded-options.patch

```diff
@@ -48,8 +50,8 @@ index 017f7cf..3d8cc9f 100644
  
  include(CheckFunctionExists)
  include(CheckIncludeFiles)
-@@ -62,10 +66,10 @@ else()
-   set(WINDOWS_CRYPTO_FOUND FALSE)
+@@ -54,10 +58,10 @@ if(WIN32)
+   endif()
  endif()
  
 -option(BUILD_SHARED_LIBS "Build shared libraries" ON)
```

Comment:
Why are we disabling these? I'd assume it work just fine with versioned shared libraries.

Reply:
It fails to compile with this enabled. It doesn't print any error.

```
make[1]: *** [lib/CMakeFiles/zip.dir/all] Error 2
make: *** [all] Error 2
```

Reply:
The error that you are actually looking for is just at the very top of the build output. Simply run the `package.sh` script a second time to only recompile the failing file, and it should be a lot clearer.

It seems to build just fine for me after removing the entire patch.